### DISCUSSION TITLE
Need help. Game with black screen.

### DISCUSSION THREAD

**Discussion by Jnills** [LostArtefacts/TRX#1105] [General]
Need help. Until version T1M 2.16, the game opened and worked normally. I didn't install new updates for a long time and I formatted my computer. Now with TRX 3.0.4 all I get when I open the game is a black screen. I tested different settings and only when deactivating the function that enables the EIDOS logo do the fmvs appear, however the menu is again with a black screen playing the theme song and soon after the game stops responding.

For more details I can inform my operating system to see if I am dealing with incompatibilities.

**Comment by lahm86** [comment]
Do you have multiple GPUs? This sounds like Windows is picking the wrong one for the exe. There are steps on the page below to allow you to force it to use your high performance GPU (it mentions Win10 but it applies to Win11 as well).

https://www.howtogeek.com/351522/how-to-choose-which-gpu-a-game-uses-on-windows-10/

**Reply by Jnills** [reply]
I only use my laptop's integrated GPU. Until version 2.16, I never had any problems playing the game. My operating system was Windows 10, version 1507 (from 2015). I formatted and maintained this same version of O.S. 1507.
For the first time I installed version TR1X 3.0.4 and strangely the black screen occurred.

I decided to update the operating system to the latest version and the game started working again. Although I prefer the old version of O.S. that I used.

I just didn't understand if the problem was the .exe update that became incompatible with the OS version I was using, or if it was due to the formatting, despite having updated the drivers, perhaps there was a missing component to run the game.

Thank you for your attention.
